modules/overview-of-model-registries.adoc

```diff
@@ -27,23 +27,31 @@ A model registry is an important component in the lifecycle of an artificial int
 
 Model registries provide a structured and organized way to store, share, version, deploy, and track models. 
 
+ifdef::upstream[]
+To use model registries in {productname-short}, an {openshift-platform} cluster administrator must configure the model registry component in {openshift-platform}. For more information, see link:{odhdocshome}/working-with-model-registries/#configuring-the-model-registry-component_model-registry[Configuring the model registry component].
+
+After the model registry component is configured, an {productname-short} administrator can create model registries in {productname-short} and grant model registry access to the data scientists that will work with them. For more information, see link:{odhdocshome}/working-with-model-registries/#creating-a-model-registry_model-registry[Creating a model registry] and link:{odhdocshome}/working-with-model-registries/#managing-model-registry-permissions_model-registry[Managing model registry permissions].
+
+Data scientists with access to a model registry can store, share, version, deploy, and track models using the model registry feature. For more information, see link:{odhdocshome}/working-with-model-registries/#working-with-model-registries_model-registry[Working with model registries]. 
+endif::[]
+
 ifndef::upstream[]
 
 ifdef::self-managed[]
-To use model registries in {productname-short}, an OpenShift cluster administrator must configure the model registry component in OpenShift. For more information, see link:{rhoaidocshome}{default-format-url}/configuring_the_model_registry_component/configuring-the-model-registry-component_model-registry-config[Configuring the model registry component].
+To use model registries in {productname-short}, an {openshift-platform} cluster administrator must configure the model registry component in {openshift-platform}. For more information, see link:{rhoaidocshome}{default-format-url}/configuring_the_model_registry_component/configuring-the-model-registry-component_model-registry-config[Configuring the model registry component].
 
-After the model registry component is configured, an {productname-short} administrator can create model registries in {productname-short} and grant model registry access to the data scientists that will work with them. For more information, see link:{rhoaidocshome}{default-format-url}/managing_model_registries/index[Managing model registries].
+After the model registry component is configured, an {productname-short} administrator can create model registries in {productname-short} and grant model registry access to the data scientists that will work with them. For more information, see link:{rhoaidocshome}{default-format-url}/managing_model_registries[Managing model registries].
```

Comment:
Also for future consideration: If the supporting text is identical, it is usually not necessary to have separate conditionalised sections for links to SM and cloud - the attribute values in the link URL will change instead.

(Exception: For links to sections in the installation guide, we use separate conditionalised sections, so that we can add a reference to the disconnected guide in the SM case.)